ui/raidboss/data/02-arr/dungeon/brayfloxs_longstop.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.BrayfloxsLongstop,
+  triggers: [
+    {
+      // Numbing Breath is Stunnable
+      id: 'Brayflox Normal Numbing Breath',
+      netRegex: NetRegexes.startsUsing({ id: '1FA', source: 'Great Yellow Pelican' }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      // Pelican Adds
+      id: 'Brayflox Normal Pelican Adds',
+      netRegex: NetRegexes.message({ line: 'A flock of ziz emerges from the jungle!', capture: false }),

```

Comment:
Is it possible to use an addedCombatant trigger here rather than matching against game logs?

Reply:
The NPC Name IDs for the adds are `['1281', '1282', 1283']`. (Blue Back, Violet Back, and Sable Back.) You will want to check for any one of them and include `suppressSeconds`, since there are packs that don't include all three.